pkg/asset/rhcos/image.go

```diff
@@ -63,6 +63,8 @@ func (i *Image) Generate(p asset.Parents) error {
 }
 
 func osImage(config *types.InstallConfig) (string, error) {
+	arch := config.ControlPlane.Architecture
```

Comment:
It's not obvious to me why we'd want to use the control-plane choice here.  Should we be passing in the arch from the caller?  Should we have an argument to select the role?

Reply:
Ah, because we don't support heterogenous clusters yet.  I'm fine punting on this until we do.

Reply:
I guess I could have passed it through as an argument. Is that what you are asking?